src/OpenTelemetry.Exporter.Jaeger/ApacheThrift/Protocol/TCompactProtocol.cs

```diff
@@ -1,4 +1,4 @@
-﻿// <auto-generated/> (Turns off StyleCop analysis in this file.)
+// <auto-generated/> (Turns off StyleCop analysis in this file.)
```

Comment:
Q.: Are this and the other files really auto-generated? The comment in parenthesis makes me question that...

Reply:
These files are copied from [ApacheThrift](https://github.com/apache/thrift/tree/master/lib/netstd/Thrift). Not really auto-generated, that line just turns off stylecop in the files since ApacheThrift uses different rules.

/cc @eddynaka